connectors/src/connectors/confluence/index.ts

```diff
@@ -56,11 +57,13 @@ export class ConfluenceConnectorManager extends BaseConnectorManager<null> {
   }: {
     dataSourceConfig: DataSourceConfig;
     connectionId: string;
-  }): Promise<Result<string, Error>> {
+  }): Promise<Result<string, CreateConnectorError>> {
     const confluenceAccessTokenRes =
       await getConfluenceAccessToken(connectionId);
     if (confluenceAccessTokenRes.isErr()) {
-      return new Err(confluenceAccessTokenRes.error);
+      return new Err(
```

Comment:
NIT: We could have the util function returning directly the Err (instead of the Error), since we always use it as is?